operators/multiclusterobservability/api/shared/multiclusterobservability_shared.go

```diff
@@ -25,6 +25,14 @@ func (u URL) Validate() error {
 	return err
 }
 
+func (u URL) Host() (string, error) {
```

Comment:
nit but naming is confusing here since the function returns both the host and the parsed path!